### ISSUE TITLE
Lutris should look for games in the manually added Steam libraries

### ISSUE THREAD

**Issue by Wopfi** [lutris/lutris#1803]
My Steam library is located on a different disk than my home directory but when starting Lutris with
`--list-stream-folders`
is only shows the default path
`~/.local/share/Steam/SteamApps`
and my added wine Steam libraries.

It looks like Lutris does only detect Steam games in the default Steam library directory but not in those added manually.

**Comment by sparr** [comment]
I went into Steam > Settings > Downloads > Steam Library Folders > Add Library Folder and added a second steam library location, a newly created empty folder, then ran `lutris --list-steam-folders` and it was able to see that folder listed in addition to the original default location.

@Wopfi can you elaborate on the scenario that produced your failure mode here?

**Comment by strycore** [comment]
yes, this should be working already, I never use the default Steam library folder.
or this bug needs more details.

**Comment by Wopfi** [comment]
> @Wopfi can you elaborate on the scenario that produced your failure mode here?

That's strange. I currently have two library directories configured in Steam. One is the default path and the other is my second hdd where all the games are located.
When running lutris 0.5.0 or 0.5.1 with the --list-steam-folders option, the output looks like this:

`2019-03-21 18:52:10,824: Using NVIDIA drivers 390.116 for x86_64`
`2019-03-21 18:52:10,824: GPU: GeForce GTX 970`
`2019-03-21 18:52:10,824: GPU: 10DE:13C2 1458:367A using nvidia drivers`
`2019-03-21 18:52:10,846: Vulkan is supported`
`2019-03-21 18:52:10,847: Updating DXVK versions`
`/home/username/.local/share/Steam/SteamApps`
`/disks/other_data/wine/prefixes/steam64/drive_d/Steam/steamapps`
`2019-03-21 18:52:10,867: Shutting down Lutris`

The first directory shown is the default path and the second is my wine Steam library which I configured via Wine Steam > Configure > Custom Steam location. My second Steam library directory which would be _/disks/other_data/steam_library_, doesn't show up at all. I also tried to add a new library folder in steam. Without any change in lutris' output.

I'm using the Lutris Version from the official openSUSE 15.0 repository at https://software.opensuse.org/package/lutris
but also tried the newest version direct from GitHub. Both versions show the same result.

Are there any log files or any other output I can check to see what goes wrong?
